website/src/content/concepts/integrate-with-your-own-api.mdx

```diff
@@ -170,6 +201,8 @@ The Exchange token structure consist of **Registered Claim Names** and **Private
 - `iat`: This value represents the time at which the JWT was issued.
 - `type`: This value is a **private claim** and is always set to `exchange`.
 - `<iss>/claims/project_key`: This value represents the Project key that the request is associated to. The claim is a **public claim** prefixed by a collision-resistant namespace, as per the JWT specification.
```

Comment:
```suggestion
- `<iss>/claims/project_key`: Represents the Project key associated with the request. The claim is a **public claim** prefixed by a collision-resistant namespace, as per the JWT specification.
```

Reply:
Updated in this commit: c8053312999955e37e23bc1c73a9fb8896af5314